launch_testing/setup.py

```diff
@@ -11,10 +11,11 @@
     data_files=[
         ('share/ament_index/resource_index/packages', ['resource/launch_testing']),
         ('lib/launch_testing', glob.glob('example_processes/**')),
-        ('share/launch_testing/examples', glob.glob('examples/[!_]**')),
+        ('share/launch_testing/examples', glob.glob('test/launch_testing/examples/[!_]**')),
     ],
     entry_points={
-        'console_scripts': ['launch_test=launch_testing.launch_test:main']
+        'console_scripts': ['launch_test=launch_testing.launch_test:main'],
+        'pytest11': ['launch = launch_testing.pytest.hooks'],
```

Comment:
This registers the marker `launch` but the tests use the marker `launch_test` everywhere?

Reply:
Launch tests that are run using `pytest` do, or otherwise it wouldn't work. We have many launch tests in other repositories that still run outside `pytest`. I haven't changed those.